pkg/pagerduty/service.go

```diff
@@ -141,25 +163,37 @@ func (data *Data) CreateService() (string, error) {
 		EscalationPolicy:       *escalationPolicy,
 		AutoResolveTimeout:     &data.autoResolveTimeout,
 		AcknowledgementTimeout: &data.acknowledgeTimeOut,
+		AlertCreation:          "create_alerts_and_incidents",
 	}
 
 	newSvc, err := client.CreateService(clusterService)
 	if err != nil {
 		return "", err
 	}
 
-	return newSvc.ID, nil
+	clusterIntegration := pdApi.Integration{
+		Name: "Test",
```

Comment:
Surely this isn't intended? Or is this PR a WIP?

Reply:
I had left that there while I try and figure out a better name, no idea what to call it.